app/assets/stylesheets/custom.css.scss

```diff
@@ -135,6 +130,18 @@ h1 {
   background-color: darkgrey;
 }
 
+#flash_twitter {
+  font-family: monospace;
```

Comment:
Properties should be ordered background-color, border-radius, color, font-family, margin-right, opacity, padding, text-align, text-style